cluster/src/main/java/com/hartwig/pipeline/tertiary/orange/Orange.java

```diff
@@ -0,0 +1,246 @@
+package com.hartwig.pipeline.tertiary.orange;
+
+import java.util.List;
+
+import com.hartwig.pipeline.Arguments;
+import com.hartwig.pipeline.ResultsDirectory;
+import com.hartwig.pipeline.calling.sage.SageOutput;
+import com.hartwig.pipeline.datatypes.DataType;
+import com.hartwig.pipeline.execution.PipelineStatus;
+import com.hartwig.pipeline.execution.vm.BashCommand;
+import com.hartwig.pipeline.execution.vm.BashStartupScript;
+import com.hartwig.pipeline.execution.vm.InputDownload;
+import com.hartwig.pipeline.execution.vm.VirtualMachineJobDefinition;
+import com.hartwig.pipeline.execution.vm.VirtualMachinePerformanceProfile;
+import com.hartwig.pipeline.execution.vm.VmDirectories;
+import com.hartwig.pipeline.execution.vm.java.JavaJarCommand;
+import com.hartwig.pipeline.flagstat.FlagstatOutput;
+import com.hartwig.pipeline.metadata.AddDatatype;
+import com.hartwig.pipeline.metadata.ArchivePath;
+import com.hartwig.pipeline.metadata.SomaticRunMetadata;
+import com.hartwig.pipeline.metrics.BamMetricsOutput;
+import com.hartwig.pipeline.report.EntireOutputComponent;
+import com.hartwig.pipeline.report.Folder;
+import com.hartwig.pipeline.report.RunLogComponent;
+import com.hartwig.pipeline.reruns.PersistedDataset;
+import com.hartwig.pipeline.reruns.PersistedLocations;
+import com.hartwig.pipeline.resource.ResourceFiles;
+import com.hartwig.pipeline.stages.Stage;
+import com.hartwig.pipeline.storage.GoogleStorageLocation;
+import com.hartwig.pipeline.storage.RuntimeBucket;
+import com.hartwig.pipeline.tertiary.chord.ChordOutput;
+import com.hartwig.pipeline.tertiary.cuppa.CuppaOutput;
+import com.hartwig.pipeline.tertiary.linx.LinxOutput;
+import com.hartwig.pipeline.tertiary.peach.PeachOutput;
+import com.hartwig.pipeline.tertiary.protect.ProtectOutput;
+import com.hartwig.pipeline.tertiary.purple.PurpleOutput;
+import com.hartwig.pipeline.tertiary.virus.VirusOutput;
+import com.hartwig.pipeline.tools.Versions;
+
+import org.jetbrains.annotations.NotNull;
+
+public class Orange implements Stage<OrangeOutput, SomaticRunMetadata> {
+    public static final String NAMESPACE = "orange";
```

Comment:
I think this should be `private` as we have an accessor method.

Reply:
Actually it is also referenced in a static context, but we can make it package private